packages/core-modules/contracts/modules/FeatureFlagModule.sol

```diff
@@ -0,0 +1,58 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@synthetixio/core-contracts/contracts/ownership/OwnableStorage.sol";
+import "../storage/FeatureFlag.sol";
+
+import "../interfaces/IFeatureFlag.sol";
+
+contract FeatureFlagModule is IFeatureFlag {
+    using SetUtil for SetUtil.AddressSet;
+
+    event FeatureFlagSet(bytes32 feature, bool value);
+    event FeatureFlagAddressAdded(bytes32 feature, address account);
+    event FeatureFlagAddressRemoved(bytes32 feature, address account);
+
+    function setFeatureFlag(bytes32 feature, bool value) external override {
+        OwnableStorage.onlyOwner();
+        FeatureFlag.load().featureFlags[feature].enabled = value;
+
+        emit FeatureFlagSet(feature, value);
+    }
+
+    function addToFeatureFlag(bytes32 feature, address permissioned) external override {
+        OwnableStorage.onlyOwner();
+        FeatureFlag.load().featureFlags[feature].permissionedAddresses.add(permissioned);
+
+        emit FeatureFlagAddressAdded(feature, permissioned);
+    }
+
+    function addMultipleToFeatureFlag(bytes32 feature, address[] calldata permissioned) external override {
```

Comment:
in the interest of streamlining the contract ABI, `addMultipleToFeatureFlag` and `removemultipleFromFeatureFlag` should be removed in favor of simply using `MulticallModule` with multiple `addToFeatureFlag` calls.

Reply:
@ajsantander requested i add these as a convenience function.

Reply:
@ajsantander can we get some documentation here as to the reasoning?

Reply:
i think the idea was just gas considerations since we eliminate the `CALL` ops with multicall

Reply:
1. this function is infrequently called so the amounto f gas saved lifetime is almost certainly <10m
2. the small cost of having increased gas is much lower than the significant cost of having additional surface area which we have to test, verify, and maintain

Reply:
@dbeal-eth yeah good points; and after talking to noah and afif, we're planning on like adding one or two addresses here and there, and then just removing the flag altogether to allow everyone so I think its safe to just remove these.  lemme know if you think otherwise @ajsantander